### DISCUSSION TITLE
在Excel中通过手势触发的快捷键没有正常执行

### DISCUSSION THREAD

**Discussion by huxuyf** [shuax/MouseInc.Settings#163] [Q&A]
`{
		"IgnoreGlobal": false,
		"List": [{
			"Actions": [
				["SendKeys", "Ctrl+Right"]
			],
			"Name": "最右",
			"Sign": "RIGHT",
			"Valid": true
		}],
		"Match": ["excel.exe", "et.exe"],
		"Name": "Excel"
	}],`

手势：向右划，对应的快捷键为：Ctrl+Right，效果应该是选中最右侧的单元格，但实际效果却是向右移动一格。

**Comment by shuax** [comment]
office软件有时候发送案件会失效，我也不知道为什么
